### ISSUE TITLE
Code for server.js in the example project strcture is absent.

### ISSUE THREAD

**Issue by midnightcodr** [agenda/agenda#565]
I am trying to follow https://github.com/agenda/agenda#example-project-structure but I am stuck on how to write `server.js` as the example doesn't show its content. Can someone give me some hint? Thanks.

**Comment by rconstantine** [comment]
Yeah, the docs are pretty sparse. Hard to follow. Would help if there were a complete example, maybe including difficult cases too.

**Comment by AlexanderProd** [comment]
I'd like to see a better example as well. Hopefully someone can include the contents of the `server.js` file in the example folder structure.
